.travis.yml

```diff
@@ -27,21 +27,59 @@ notifications:
 
 jobs:
   include:
+    - name: "Documentation Update"
+      script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then ./.travis/travis-docs.sh; fi'
+      arch: amd64
+
+
+    - name: "CentOS 7"
+      script: ./.travis/travis-centos.sh
+      arch: amd64
+
+
     - name: "Fedora 29"
       script: ./.travis/travis-fedora.sh
+      arch: amd64
+
     - name: "Fedora 30"
       script: ./.travis/travis-fedora.sh
+      arch: amd64
+
+    - name: "Fedora 31"
+      script: ./.travis/travis-fedora.sh
+      arch: amd64
+
     - name: "Fedora rawhide"
       script: ./.travis/travis-fedora.sh
-    - name: "CentOS 7"
-      script: ./.travis/travis-centos.sh
-    - name: "Documentation Update"
-      script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then ./.travis/travis-docs.sh; fi'
+      arch: amd64
+
+
+    - name: "Fedora 29"
+      script: ./.travis/travis-fedora.sh
+      arch: arm64
+    - name: "Fedora 30"
+      script: ./.travis/travis-fedora.sh
+      arch: arm64
+    - name: "Fedora 31"
+      script: ./.travis/travis-fedora.sh
+      arch: arm64
+    - name: "Fedora rawhide"
+      script: ./.travis/travis-fedora.sh
+      arch: arm64
+
+
     - name: "Coverity Scan"
       script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then ./.travis/travis-coverity.sh; fi'
+
```

Comment:
Should the Coverity scan get an explicit `arch: amd64`, too?

Reply:
Yes, I missed that.